rllib/offline/offline_data.py

```diff
@@ -18,25 +18,23 @@
 
 logger = logging.getLogger(__name__)
 
-# TODO (simon): Implement schema mapping for users, i.e. user define
-# which row name to map to which default schema name below.
-SCHEMA = [
-    Columns.EPS_ID,
-    Columns.AGENT_ID,
-    Columns.MODULE_ID,
-    Columns.OBS,
-    Columns.ACTIONS,
-    Columns.REWARDS,
-    Columns.INFOS,
-    Columns.NEXT_OBS,
-    Columns.TERMINATEDS,
-    Columns.TRUNCATEDS,
-    Columns.T,
+SCHEMA = {
```

Comment:
nit: Add a comment here explaining that these are the default mappings and if users define some of these keys in their custom schema, then only those column mappings will be overridden.